tests/basic/tests.py

```diff
@@ -284,6 +284,12 @@ def test_field_ordering(self):
         self.assertIsNotNone(f1)
         self.assertNotIn(f2, (None, 1, ''))
 
+    def test_field_str(self):
```

Comment:
I'd put this in `tests/model_fields/tests.py` (In fact, `test_field_str` already exist in that file). I guess the above test could also be moved (in a separate commit).